activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -38,6 +38,7 @@ def data_source_exists?(name)
       end
 
       # Returns an array of table names defined in the database.
+      # Returning tables only or including views depends on each adapters.
```

Comment:
I don't think it's worth to document this. It's been like this for a long time. I'd keep the docs for this method as is.